src/HealthChecks/Abstractions/src/HealthReport.cs

```diff
@@ -66,13 +66,13 @@ private static HealthStatus CalculateAggregateStatus(IEnumerable<HealthReportEnt
             if (currentValue > entry.Status)
             {
                 currentValue = entry.Status;
-            }
 
-            if (currentValue == HealthStatus.Unhealthy)
-            {
-                // Game over, man! Game over!
-                // (We hit the worst possible status, so there's no need to keep iterating)
-                return currentValue;
+                if (currentValue == HealthStatus.Unhealthy)
+                {
+                    // Game over, man! Game over!
```

Comment:
Does it worth considering replacing this comment with something more appropriate?